go.mod

```diff
@@ -3,77 +3,78 @@ module github.com/scribd/go-sdk
 go 1.17
 
 require (
```

Comment:
[req]: probably these changes for `go.*` files are slipped in the changeset :) Don't think we need them in this PR 🤔